src/status_im2/navigation/screens.cljs

```diff
@@ -53,7 +53,8 @@
 
     {:name      :chat
      :options   {:insets     {:top? true}
-                 :popGesture false}
+                 :popGesture false
+                 :layout     {:orientation ["portrait"]}}
```

Comment:
does that mean when we use options, they are overriding and not merged? can we find proof from the RNN library?